Sources/iOSClientExposure/Extensions/UIDevice+Extensions.swift

```diff
@@ -22,4 +22,80 @@ extension UIDevice {
         if systemName == "iPhone OS" { return "iOS" }
         return systemName
     }
+    
+    var modelName: String {
```

Comment:
un-used?

Reply:
Yes it is unused , I will remove this & refactor the extension